src/components/CreateProfileBtn/CreateProfileBtn.tsx

```diff
@@ -16,7 +15,7 @@ const CreateProfileBtn = () => {
   }
 
   const onClickHandler = async () => {
-    if (!session) {
+    if (!session?.user?.id) {
```

Comment:
**nitpick:** We should leave as it is, and make session null if it's not present